v2/internal/controllers/samples_test.go

```diff
@@ -187,10 +189,11 @@ func findRefsAndCreateSecrets(tc *testcommon.KubePerTestContext, resources []cli
 }
 
 func getTestName(group string, version string) string {
+	caser := cases.Title(language.English)
```

Comment:
I think this change may have broken tests, as I see the samples tests for some reason think they need a SUBSCRIPTION_ID, which means they thought they were recording rather than replaying.  I think that can happen if the test didn't find the file it was looking for (which is based on testName presumably defined here)

Reply:
Fixed in PR #4236